examples/service_with_ldap-user-store/service_with_ldap_user_store.client.out

```diff
@@ -0,0 +1,13 @@
+# Since `admin` has `test` the invocation will succeeded.
+$ curl -kv https://localhost:9090/ldapAuth/resourceOne -H "Authorization: Basic YWRtaW46YmFsbGVyaW5h"
+Hello, World!
+
+# Since the password for the `admin` is not correct. Authentication will not be successful.
+$ curl -kv https://localhost:9090/ldapAuth/resourceOne -H "Authorization: Basic dmlqaXRoYTpiYWxsZXJpbmE"
+Authentication failure
+
+# Since resource configuration has overridden the required scopes to denote that
```

Comment:
Change to:

# Resource configuration has overridden the required scopes to denote that
# `test` is required to invoke `resourceTwo` resource. Hence authorization will
# not be successful.